Index: cpudynd.c
===================================================================
--- cpudynd.c
+++ cpudynd.c	2011-05-08 12:54:41.843766964 +0200
@@ -226,7 +226,7 @@
 		} else if (!strcmp(argv[i], "-X")) {
 			do_exit = 1;
 		} else if ((!strcmp(argv[i], "-?"))||(!strcmp(argv[i], "--help"))) {
-			printf( str_usage );
+			printf( "%s", str_usage );
 			exit(0);
 		} else if (!strcmp(argv[i], "-asus")) {
 			asus_disable = 0;
